src/internal/functions/Get-ReadmeContent.ps1

```diff
@@ -0,0 +1,156 @@
+The Resource API ships inside of Puppet starting with version 6.
+While it is _technically_ possible to add the Resource API functionality to Puppet 5.5.x, the DSC functionality has _not_ been tested in this setup.
+For more information on the Resource API, review the [documentation]($ResourceApiDocumentation).
+
+The module also depends on the `pwshlib` module.
+This Puppet module includes two important things: the ruby-pwsh library for running PowerShell code from ruby and the base provider for DSC resources, which this module leverages.
+
+All of the actual work being done to call the DSC resources vendored with this module is in [this file]($BaseProviderSource) from the `pwshlib` module.
+This is important for troubleshooting and bug reporting, but doesn't impact your use of the module except that nothing will work if this dependency isn't installed alongside this module!
+
+## Usage
+
+You can specify any of the DSC resources from this module like a normal Puppet resource in your manifests.
+The examples below use DSC resources from from the [`PowerShellGet`]($PowerShellGetUri) repository, regardless of what module you're looking at here;
+the syntax, not the specifics, is what's important.
+
+For reference documentation about the DSC resources exposed in this module, see the [Reference](REFERENCE.md) file.
+
+``````puppet
+# Include a meaningful title for your resource declaration
+dsc_psrepository { 'Add team module repo':
+    dsc_name               => 'foo',
+    # Note that we specify dsc_ensure; do NOT specify the Puppet
+    # ensure property, it exists only for the underlying system.
+    # You will always use dsc_ensure.
+    dsc_ensure             => present,
+    # This location is nonsense, can be any valid folder on your
+    # machine or in a share, any location the SourceLocation param
+    # for the DSC resource will accept.
+    dsc_sourcelocation     => 'C:\Program Files',
+    # You must always pass an enum fully lower-cased;
+    # Puppet is case sensitive even when PowerShell isn't
+    dsc_installationpolicy => untrusted,
+}
+
+dsc_psrepository { 'Trust public gallery':
+    dsc_name               => 'PSGallery',
+    dsc_ensure             => present,
+    dsc_installationpolicy => trusted,
+}
+
+dsc_psmodule { 'Make Ruby manageable via uru':
+  dsc_name   => 'RubyInstaller',
+  dsc_ensure => present,
+}
+``````
+
+For more information about using a built module, check out our [narrative documentation]($NarrativeDocumentation).
+
+## Troubleshooting
+
+<!-- TODO: move all generic docs into main PuppetDscBuilder README and link from here; minimize content that we would need to update! ->
+
+In general, there are three broad categories of problems:
+
+1. Problems with the way the underlying DSC resource works.
+1. Problems with the type definition, where you can't specify a valid set of properties for the DSC resource
```

Comment:
```suggestion
2. Problems with the type definition, where you can't specify a valid set of properties for the DSC resource
```

Reply:
enumerations get automatically numbered when markdown is rendered. always using `1` allows for easier rearranging, and reduces the temptation to reference items by ephemeral numbers.

Reply:
Ah, OK - I learnt something!